packages/fxa-settings/src/components/GetDataTrio/index.tsx

```diff
@@ -54,6 +58,35 @@ export const GetDataCopySingleton = ({ value, onAction }: GetDataTrioProps) => {
   );
 };
 
+export const GetDataCopySingletonInline = ({
+  value,
+  onAction,
+}: GetDataTrioProps) => {
+  return (
+    <Localized
+      id="get-data-trio-copy-2"
+      attrs={{ title: true, ariaLabel: true }}
+    >
+      <button
+        title="Copy"
+        type="button"
+        onClick={async () => {
+          const copyValue = Array.isArray(value) ? value.join('\r\n') : value;
+          await copy(copyValue);
+          onAction?.('copy');
+        }}
+        data-testid="databutton-copy"
```

Comment:
Is it OK that this `data-testid` is a dupe of L47 above?
Not sure if that makes a difference for testing, or if we're indirectly only testing that first button if we're in fact using the testid.

Reply:
We should only have one of the two variations at any time in the page, but we want both versions to pass the same test(s), so this should be alright.